webapp/src/components/Earn/Counterparties.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styled from "styled-components";
+import { SecondaryText } from "shared/lib/designSystem";
+import CounterpartyDetail from "./CounterpartyDetail";
+
+const ParagraphText = styled(SecondaryText)`
+  color: rgba(255, 255, 255, 0.64);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+`;
+
+const Container = styled.div`
+  overflow-x: hidden;
+`;
+
+const CounterpartyList = [
+  "R-EARN DIVERSIFIED",
+  "ORTHOGONAL",
+  "ALAMEDA RESEARCH",
+  "CITADEL",
+] as const;
+
+export type Counterparty = typeof CounterpartyList[number];
+const CounterpartyListSubset: Counterparty[] = ["R-EARN DIVERSIFIED"];
+
+const Counterparties: React.FC = () => {
+  // const { vaultAccounts } = useVaultAccounts("earn");
```

Comment:
Nit: Is this still needed?

Reply:
this is wip for multi vaults added comments for this